src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -523,12 +523,19 @@ public String getQuestionResultStatisticsCsv(
         DecimalFormat df = new DecimalFormat("#.##");
         
         for(Entry<String, Integer[]> entry : optionTotalCount.entrySet() ){
-            double average = entry.getValue()[0]/entry.getValue()[1];
-            fragments += entry.getKey() + ","
-                      + df.format(average) + Const.EOL;
+        	double average = entry.getValue()[0]/entry.getValue()[1];
```

Comment:
This line should have the same indentation as the ones below. Check if you are mixing tabs and spaces.